base/applications/charmap/charmap.c

```diff
@@ -4,7 +4,7 @@
  * FILE:        base/applications/charmap/charmap.c
  * PURPOSE:     main dialog implementation
  * COPYRIGHT:   Copyright 2007 Ged Murphy <gedmurphy@reactos.org>
- *
+ *              Copyright 2022 Katayama Hirofumi MZ <katayama.hirofumi.mz@gmail.com>
```

Comment:
I think this is more justified in map.c than here!

Reply:
Moved.